### PR TITLE
Linux C1 Retagging

### PR DESCRIPTION
**Pull request by mihaiberq**

### CODE REVIEW

**Comment by enki-hq** [comment]
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/bash-environment/adding-directories-to-your-path.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/bash-environment/customizing-bash-via-bashrc.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/bash-environment/manipulating-environment-variables.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/bash-environment/understanding-the-path-environment-variable.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/bash-environment/what-are-environment-variables.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/conditions/compound-commands.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/conditions/conditional-command-execution-operator.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/conditions/else-statements-using-the-operator.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/conditions/sequential-execution-using-the-statement-separator.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/conditions/the-test-built-in.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/default-permissions/making-a-file-unalterable-with-chattr.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/default-permissions/the-setgid-permission.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/default-permissions/the-setuid-permission.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/default-permissions/the-sticky-bit-permission.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/default-permissions/understanding-umask.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management-tips/deleting-files-in-a-secure-manner.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management-tips/deleting-non-empty-directories.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management-tips/truncate-files-with-cat-instead-of-rm.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management-tips/using-touch-to-alter-files-modification-time.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management-tips/wrap-text-with-fold.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management/matching-multiple-files-with-wildcards.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management/the-directory-tree.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management/using-find-to-search-by-file-size.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management/using-find-to-search-by-filename.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/file-management/using-which-to-find-executables.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipeline-tools/duplicate-pipe-content-with-tee.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipeline-tools/progress-of-data-through-a-pipe-with-pv.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipeline-tools/shebang-explained.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipeline-tools/the-2-1-redirection.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipeline-tools/using-xargs-to-build-commands.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipelines/appending-output-with.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipelines/input-and-output-redirection-with-and.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipelines/the-basics-of-command-substitution.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipelines/the-basics-of-the-pipe-operator.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/pipelines/the-standard-streams-stdin-stdout-and-stderr.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/super-users-and-root/executing-commands-with-sudo-without-password.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/super-users-and-root/get-file-permissions-in-octal-form.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/super-users-and-root/granting-root-access-to-a-user.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/super-users-and-root/run-previous-command-as-root.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/super-users-and-root/su-vs-sudo.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-history/bash-history-reverse-search.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-history/clear-bash-history.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-history/execute-a-command-without-saving-it-in-history.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-history/keep-useful-commands-in-your-shell-history-with-tags.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-history/terminal-incognito-mode.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-power-user-tips/clear-the-terminal-instantly.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-power-user-tips/find-any-unix-linux-command.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-power-user-tips/quick-bash-shortcuts.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-power-user-tips/retrieve-previous-arguments.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/terminal-power-user-tips/useful-aliasing-in-bash.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management-utilities/best-password-generation-utilities.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management-utilities/fast-file-indexing-with-updatedb-and-locate.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management-utilities/find-the-unknown-process-preventing-deleting-of-files.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management-utilities/get-to-know-your-commands-with-type.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management-utilities/random-password-generator.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management/adding-new-users-with-useradd.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management/changing-user-passwords-with-passwd.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management/linux-user-management.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management/running-a-command-as-another-local-user.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-management/usermod-recipes.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-permissions/changing-file-ownership-with-chown.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-permissions/changing-file-permissions-with-chmod.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-permissions/changing-group-membership-with-usermod.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-permissions/file-ownership.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c
- [ ] https://enkipro.com/preview-insight/sagelabs/content/Linux/User%20and%20File%20Management/user-permissions/users-and-groups-on-linux.md?ref=5220288a1f76974062c0cc16c4bb58df5f17ad7c

File: Linux/User and File Management/default-permissions/the-setuid-permission.md [hunk 1]
```diff
@@ -10,17 +10,13 @@ type: normal
...
-
   - terminal
-
   - permissions
-
   - chmod
-
   - setuid
 
+
```
Comment (lizTheDeveloper) [inline Linux/User and File Management/default-permissions/the-setuid-permission.md]: @mihaiberq looks like the tagger didn't tag this insight, do you know why?

File: Linux/User and File Management/default-permissions/the-sticky-bit-permission.md [hunk 2]
```diff
@@ -10,15 +10,12 @@ type: normal
...
-
   - linux
-
   - terminal
-
   - permissions
-
   - sticky bit
 
+
```
Comment (lizTheDeveloper) [inline Linux/User and File Management/default-permissions/the-sticky-bit-permission.md]: here too, and on several more files

File: Linux/User and File Management/default-permissions/understanding-umask.md [hunk 3]
```diff
@@ -10,17 +10,13 @@ type: normal
...
-
   - permissions
-
   - umask
-
   - terminal
-
   - files
 
+
```
Comment (lizTheDeveloper) [inline Linux/User and File Management/default-permissions/understanding-umask.md]: it might be that we used 🔮  instead of 🐉  in some places?

Comment (mihaiberq) [reply Linux/User and File Management/default-permissions/understanding-umask.md]: I checked the revamp file and those insights/workouts have no tags.

**Pull request merged**